src/Acuminator/Acuminator.Analyzers/FixProviders/DAC/DacPropertyAttributes/DacExtensionDefaultAttributeFix.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Composition;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.Editing;
+using Acuminator.Utilities;
+using PX.Data;
+
+namespace Acuminator.Analyzers.FixProviders
+{
+	[Shared]
+	[ExportCodeFixProvider(LanguageNames.CSharp)]
+	public class DacExtensionDefaultAttributeFix : CodeFixProvider
+	{
+		public override ImmutableArray<string> FixableDiagnosticIds { get; } =
+			ImmutableArray.Create(Descriptors.PX1030_DefaultAttibuteToExisitingRecords.Id);
+
+		public override FixAllProvider GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
+
+		public override Task RegisterCodeFixesAsync(CodeFixContext context)
+		{
+			return Task.Run(() =>
+			{
+				var diagnostic = context.Diagnostics.FirstOrDefault(d => d.Id == Descriptors.PX1030_DefaultAttibuteToExisitingRecords.Id);
+
+				if (diagnostic == null)
```

Comment:
This code is unnecessary; this scenario is handled by VS - supported diagnostics are specified explicitly in `FixableDiagnosticIds`